src/renderer/features/assets/AssetsChainView/ui/AssetsChainView.tsx

```diff
@@ -34,26 +34,23 @@ export const AssetsChainView = ({ query, activeShards, hideZeroBalances, assetsV
     if (!activeWallet || assetsView !== AssetsListView.CHAIN_CENTRIC || !activeShards.length) return;
 
     const isMultisig = walletUtils.isMultisig(activeWallet);
+    const multisigChainToInclude = isMultisig ? activeWallet.accounts[0].chainId : undefined;
 
     const availableChains = Object.values(chains).filter((chain) => {
       return activeWallet.accounts.some((account) => {
         return (
-          activeWallet &&
           accountUtils.isNonBaseVaultAccount(account, activeWallet) &&
           accountUtils.isChainAndCryptoMatch(account, chain)
         );
       });
     });
 
     const filteredChains = availableChains.filter((c) => {
-      if (!connections[c.chainId]) {
-        return false;
-      }
+      if (!connections[c.chainId]) return false;
+      if (networkUtils.isDisabledConnection(connections[c.chainId])) return false;
```

Comment:
Check that connection exist before checking status

Reply:
`if (!connections[c.chainId]) return false;`
Or what do you mean?

Reply:
```
const connection = connections[c.chainId];
if (nullable(connection)) return false
```